free-3.md

```diff
@@ -10,41 +10,126 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Venn diagrams display data in a cool and organized way. Many programs or apps 
+already exist for drawing Venn diagrams, but the ones I found don't adjust the 
+circles according to the text within. For example, some programs make the users 
+specify the exact size of the circles and data. Thus, it takes many tries to 
+format venn diagrams exactly as envisioned. But what happens if the data 
+changes? Then people would have to repeat this struggle.
+
+I want to design a DSL that, given text for each circle (and intersection of 
+circles), creates an image rendering of the venn diagram with the text snugly 
+inside the circles. Basically, circles are auto-formatted according to the text.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+At first, I was debating if this should be an app or a language. But I think I 
+can make it a language if I break down venn diagrams into "data", "category", 
+"unions", and "intersections", and "none" (things outside all the circles). 
+Then, it's just a matter of creating the data for each category and 
+intersecting them.
+
+Thus, users can easily create data (building blocks) and connect them 
+through intersections.
```

Comment:
I think you can make this work as a language, but it does seem on the small side, and I'm not completely convinced that there is a domain that needs the language (yet).